### PR TITLE
feat: regular pings for boltz ws

### PR DESCRIPTION
**Pull request by jackstar12**

### CODE REVIEW

File: boltz/ws.go [hunk 1]
```diff
@@ -85,6 +80,31 @@ func (boltz *Websocket) Connect() error {
...
+		logger.Silly("Received pong")
+		return setDeadline()
+	})
+
+	go func() {
+		ticker := time.NewTicker(pingInterval)
+		defer ticker.Stop()
+		for range ticker.C {
+			// wont wait longer for writing then for the response
+			err := conn.WriteControl(websocket.PingMessage, []byte(pingMessage), time.Now().Add(pongWait))
```
Comment (michael1011) [inline boltz/ws.go:97]: Empty bytes in the frame do not work?

File: boltz/ws.go [hunk 2]
```diff
@@ -85,6 +80,31 @@ func (boltz *Websocket) Connect() error {
...
+	conn.SetPongHandler(func(string) error {
+		logger.Silly("Received pong")
+		return setDeadline()
+	})
+
+	go func() {
+		ticker := time.NewTicker(pingInterval)
+		defer ticker.Stop()
+		for range ticker.C {
+			// wont wait longer for writing then for the response
```
Comment (michael1011) [inline boltz/ws.go:96]:
```suggestion
			// Will not wait longer with writing than for the response
```

File: boltz/ws.go [hunk 3]
```diff
@@ -85,6 +80,31 @@ func (boltz *Websocket) Connect() error {
...
+			err := conn.WriteControl(websocket.PingMessage, []byte(pingMessage), time.Now().Add(pongWait))
+			if err != nil {
+				if boltz.closed {
+					return
+				}
+				logger.Error("could not send ping, closing conn: " + err.Error())
```
Comment (michael1011) [inline boltz/ws.go:102]:
```suggestion
				logger.Error("could not send ping; closing connection: %s", err)
```

File: boltz/ws.go (hunk 1 above)
Comment (jackstar12) [reply boltz/ws.go]: It does not

**Pull request merged**